mesa/space.py

```diff
@@ -989,22 +986,20 @@ def is_cell_empty(self, node_id: int) -> bool:
         return not self.G.nodes[node_id]["agent"]
 
     def get_cell_list_contents(self, cell_list: list[int]) -> list[GridContent]:
-        """Returns the contents of a list of cells ((x,y) tuples)
-        Note: this method returns a list of `Agent`'s; `None` contents are excluded.
+        """Returns the agents in a list of nodes; nodes with empty content are excluded.
+        Note: this method returns a list of `Agent`'s.
         """
-        return list(self.iter_cell_list_contents(cell_list))
-
-    def get_all_cell_contents(self) -> list[GridContent]:
-        """Returns a list of the contents of the cells
-        identified in cell_list."""
-        return list(self.iter_cell_list_contents(self.G))
-
-    def iter_cell_list_contents(self, cell_list: list[int]) -> list[GridContent]:
-        """Returns an iterator of the contents of the cells
-        identified in cell_list."""
         list_of_lists = [
             self.G.nodes[node_id]["agent"]
             for node_id in cell_list
             if not self.is_cell_empty(node_id)
         ]
         return [item for sublist in list_of_lists for item in sublist]
+
+    def get_all_cell_contents(self) -> list[GridContent]:
+        """Returns a list of all the agents in the network."""
+        return self.get_cell_list_contents(self.G)
+
+    def iter_cell_list_contents(self, cell_list: list[int]) -> Iterator[GridContent]:
```

Comment:
Given that falsy contents are not included, the annotation should have been `Iterator[Agent]`. And in other cases it should have been `Iterator[MultiGridContent]`, because technically `NetworkGrid` is a multi grid, due to its content being a list. But this should be fixed in a separate PR.

Reply:
I agree to address all of this in a separate PR,  good to have a reminder here, there is something else to do to merge this one instead? :-)